apps/src/code-studio/pd/form_components/FormController.jsx

```diff
@@ -213,6 +221,20 @@ export default class FormController extends React.Component {
   }
 
   /**
+   * switch to the specified page in sequence, if it exists
+   */
+  setPage(i) {
+    const page = Math.min(
```

Comment:
If the page doesn't exist, does it make sense to log an error so we can see this in newRelic?